runtime/doc/options.txt

```diff
@@ -5560,7 +5560,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	old Vi scripts.
 
 						*'renderoptions'* *'rop'*
-'renderoptions' 'rop'	Removed. {Nvim} will choose the best renderer available.
+'renderoptions' 'rop'	Removed. {Nvim}
```

Comment:
We added this line intentionally to communicate that Neovim doesn't need this option. Maybe the wording can be improved, idk.